ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -34,33 +36,63 @@ const StyledVariantRow = styled(Grid.Row)`
   }}
 `
 
-const Variant = ({ variant }) => {
+const StyledCompoundHetLink = styled(Grid.Column)`
+  margin-right: -1em !important;
+`
+
+const StyledCompoundHetRows = styled(Grid)`
+  margin-left: 0em !important;
+  margin-right: 1em !important;
+  margin-top: 0.5em !important;
+  margin-bottom: 0 !important;
+`
+
+const Variant = ({ variant, isCompoundHet }) => {
   const mainGeneId = getVariantMainGeneId(variant)
   return (
-    <StyledVariantRow severity={CLINSIG_SEVERITY[(variant.clinvar.clinicalSignificance || '').toLowerCase()]}>
+    <StyledVariantRow key={variant.variantId} severity={CLINSIG_SEVERITY[(variant.clinvar.clinicalSignificance || '').toLowerCase()]} iscompoundhet={(isCompoundHet || false).toString()} >
+      {isCompoundHet &&
+        <StyledCompoundHetLink width={16}>
+          {variant.familyGuids.map(familyGuid =>
+            <FamilyVariantTags familyGuid={familyGuid} variant={variant} key={variant.variantId} isCompoundHet />,
+          )}
+        </StyledCompoundHetLink>
+      }
       <Grid.Column width={16}>
         <Pathogenicity variant={variant} />
       </Grid.Column>
-      {variant.familyGuids.map(familyGuid =>
+      {!isCompoundHet && variant.familyGuids.map(familyGuid =>
         <Grid.Column key={familyGuid} width={16}>
           <FamilyVariantTags familyGuid={familyGuid} variant={variant} />
         </Grid.Column>,
       )}
+      {isCompoundHet &&
+      <Grid.Column>
+        {Object.keys(variant.transcripts).filter(geneId => geneId !== mainGeneId).map(geneId =>
+          <VariantGene key={geneId} geneId={geneId} variant={variant} compact />,
+        )}
+        {Object.keys(variant.transcripts).length > 1 && <VerticalSpacer height={20} />}
+        {variant.familyGuids.map(familyGuid =>
```

Comment:
for things like this where you end up copying the same code twice so you can put it in a different place for compound hets you should assign it to a const once at the top and then reference it in multiple places. So you would have `const variantIndividuals = ...` up where `mainGeneId` is defined and then here all you would have is `{variantIndividuals}`